src/components/SetupBanner.tsx

```diff
@@ -223,6 +249,42 @@ export function SetupBanner() {
                     handleNodeInstallClick={handleNodeInstallClick}
                     finishNodeInstall={finishNodeInstall}
                   />
+
+                  <div className="mt-3 pt-3 border-t border-gray-200 dark:border-gray-700">
+                    <button
+                      onClick={() => setShowManualConfig(!showManualConfig)}
+                      className="text-sm text-blue-600 dark:text-blue-400 hover:underline"
+                    >
+                      Node.js already installed? Configure path manually →
+                    </button>
+
+                    {showManualConfig && (
+                      <div className="mt-3 p-3 bg-gray-50 dark:bg-gray-800 rounded-lg">
+                        <p className="text-sm text-gray-700 dark:text-gray-300 mb-3">
+                          Select the folder where Node.js is installed (usually
```

Comment:
I think I would just remove this parenthetical comment, since I think people who _already_ have Node.js installed probably know this and if they don't, we can make it clearer in the error message (e.g. did not find "node.exe" at x/y/z path) - see other comment